external/mmdetection/detection_tasks/apis/detection/openvino_task.py

```diff
@@ -395,7 +395,7 @@ def infer(
             feature_vector, saliency_map = features
             if feature_vector is not None:
                 representation_vector = TensorEntity(
-                    name="representation_vector", numpy=feature_vector.reshape(-1)
+                    name="representation_vector", numpy=np.copy(feature_vector.reshape(-1))
```

Comment:
Should we copy `saliency_map` as well?

Reply:
I checked and it doesn't require copying